Packs/WALLIXBastion/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,70 @@
+
+#### Integrations
+
+##### WALLIX Bastion
+- Updated the Docker image to: *demisto/python3:3.11.10.111526*.
+
+
+- The integration now returns the ID of created objects on the Bastion
+
+- Added support for nullable arguments
+
+- Added new commands:
+    - wab-add-connection-policy
+    - wab-add-mapping-in-domain
+    - wab-add-password-change-policy
+    - wab-add-restriction-to-targetgroup
+    - wab-add-restriction-to-usergroup
+    - wab-add-timeframe
+    - wab-change-password-or-ssh-key-of-account
+    - wab-delete-connection-policy
+    - wab-delete-mapping-of-domain
+    - wab-delete-password-change-policy
+    - wab-delete-restriction-from-targetgroup
+    - wab-delete-restriction-from-usergroup
+    - wab-delete-scan
+    - wab-delete-timeframe
+    - wab-edit-connection-policy
+    - wab-edit-mapping-of-domain
+    - wab-edit-mappings-of-domain
+    - wab-edit-password-change-policy
+    - wab-edit-restriction-from-targetgroup
+    - wab-edit-restriction-from-usergroup
+    - wab-edit-scan
+    - wab-edit-timeframe
+    - wab-edit-user
+    - wab-get-cluster
+    - wab-get-clusters
+    - wab-get-connection-policies
+    - wab-get-connection-policy
+    - wab-get-external-authentication-group-mappings
+    - wab-get-local-domain-data-for-application
+    - wab-get-local-domain-of-device
+    - wab-get-local-domains-data-for-application
+    - wab-get-local-domains-of-device
+    - wab-get-mapping-of-domain
+    - wab-get-mappings-of-domain
+    - wab-get-target-group-restriction
+    - wab-get-user-group-restriction
+    - wab-get-password-change-policies
+    - wab-get-password-change-policy
+    - wab-get-passwordrights
+    - wab-get-passwordrights-user-name
+    - wab-get-rest-api
+    - wab-get-target-group-restrictions
+    - wab-get-timeframe
+    - wab-get-timeframes
+    - wab-get-user-group-restrictions
+
+- **Breaking changes:** removed deprecated command wab-get-metadata-of-one-or-multiple-sessions
+
+- **Breaking changes:** removed deprecated arguments in:
+    - wab-edit-application: application_put__meters
+
+- **Breaking changes:** removed deprecated outputs in:
+    - wab-get-profile(s): WAB.profile_get.gui_features.dashboards
+    - wab-get-application(s)/wab-get-device(s): and wab-get-profiles: WAB.(...).tags.id
+
+- added corresponding API endpoint URL to the commands' description
```

Comment:
is it really necessary?

Reply:
The name of the commands is generated from the endpoints description, but this can be slightly confusing sometimes when two resources are closely related. Seeing the actual url which will be called helps ensure we are interacting with the expected resource and helps linking the integration commands to the API documentation available on the Bastion. We think it might be handy in some cases.

Reply:
Our code standard is to remove this url from the description.
Would appreciate if you rethink about it, in most cases the user is not aware of the endpoint which is being used so it is not relevant for them in our opinion.
Thank you!